scaling/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,34 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/http-jvm .
```

Comment:
`http-jvm` .... copy&paste from http module

Do we need Dockerfiles at all here?

Reply:
No ass far as I can tell, so I removed them

Reply:
I have kept the `Dockerfile` in all test applications, because we wanted to test them as well, somehow. (Not sure if that's gonna be possible, but that was the intent.) Indeed they are all copied from `http`, but that doesn't matter, because this is just a comment.